api/HTMLInputElement.json

```diff
@@ -694,10 +694,10 @@
               "version_added": null
             },
             "safari": {
-              "version_added": true
+              "version_added": "5.1"
```

Comment:
http://mdn-bcd-collector.appspot.com/tests/api/HTMLInputElement/formEnctype uses an instance, so I trust this.